crates/wash-cli/src/app/mod.rs

```diff
@@ -117,6 +120,16 @@ pub struct GetCommand {
     opts: CliConnectionOpts,
 }
 
+#[derive(Args, Debug, Clone)]
+pub struct StatusCommand {
+    /// The name of the app spec
+    #[clap(name = "name")]
+    model_name: String,
+
+    #[clap(flatten)]
```

Comment:
To address the overwide output problem, maybe we should introduce some sort of flattened options that represent output formatting? Similar to `CliConnectionOpts`. Unfortunately it'd have to flow through the command to the output function (`status_table`) but it might be reasonable to truncate by default (so people get a sense of the command and show the whole thing. 

Definitely can be done in a follow up though or if someone finds it really troublesome -- AFAIK we don't do precise screen width checking to ensure we don't overflow in the first place... IMO better to move fast and overflow for now then go back in and add by-default truncation with the option to print everything.

Reply:
Agreed, more smartness/optionality later is better. So long as the table columns make sense, I'm happy to ship this